cmake/define-test.cmake

```diff
@@ -103,4 +103,31 @@ function(define_test)
         ${WRAPPERS}
         partition-alloc)
     add_dependencies(check-ia2 ${MAIN})
+
+    # Find libc
+    list(APPEND LIBC_PATHS /lib/libc.so.6 /lib64/libc.so.6 /lib/x86_64-linux-gnu/libc.so.6)
```

Comment:
`find_library(LIBC_PATH libc)`?

Reply:
This (specifically, `find_library(LIBC c REQUIRED)`) doesn't work, because `libc.so` itself is a linker script with glibc, and we need the actual ELF shared object so we can modify its program headers.

Reply:
I'll add a comment explaining the workaround.